projects/netgrif-components-core/src/lib/forms/login/login-sso/abstract-login-sso.component.ts

```
@@ -0,0 +1,108 @@
+        protected _config: ConfigurationService,
+        protected _http: HttpClient,
+        protected _snackbar: SnackBarService,
+        protected _log: LoggerService,
+        protected _router: Router,
+        protected _activeRouter: ActivatedRoute,
+        protected _translate: TranslateService
+    ) {
+        this._ssoConfig = this._config.getConfigurationSubtree(['providers', 'auth', 'sso']);
+        this.loading = new LoadingEmitter();
+        this._activeRouter.queryParams.subscribe((params) => {
+            if (!!params.code) {
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)